{
  "pr_number": 733,
  "title": "Fixed NPE getting Stack Trace if Throwable is null",
  "state": "closed",
  "created_at": "2021-03-20T06:13:12Z",
  "merge_commit_sha": "1926e35e3e1e94a43c64cb4d65f51d115fae02cd",
  "base_sha": "cd29ff77783bb870dae96c2e2880a7bbbc9e61d5",
  "head_sha": "57ea02280699fc050dd8bc40f99b950de436ba38",
  "user_login": "arturobernalg",
  "extracted_count": 1,
  "skipped_count": 1,
  "files": [
    {
      "source_filename": "src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java",
      "status_in_pr": "modified",
      "do_experiment": true,
      "reason": null,
      "status": "written",
      "out_test_path": "pr_733/src/test/java/org/apache/commons/lang3/exception/ExceptionUtilsLLM_Test.java",
      "package": "org.apache.commons.lang3.exception",
      "original_class_name": "ExceptionUtilsTest2",
      "final_class_name": "ExceptionUtilsLLM_Test"
    },
    {
      "source_filename": "src/test/java/org/apache/commons/lang3/exception/ContextedExceptionTest.java",
      "status_in_pr": "modified",
      "do_experiment": false,
      "reason": "do_experiment is not true",
      "status": "skipped",
      "out_test_path": null,
      "package": null,
      "original_class_name": null,
      "final_class_name": null
    }
  ]
}